src/index.rs

```diff
@@ -125,17 +125,39 @@ pub fn row(input: &Array, row_num: u64) -> Result<Array, AfError> {
                     ,Seq::new(0.0, dims[1] as f64 - 1.0, 1.0)])
 }
 
+pub fn rows(input: &Array, first: u64, last: u64) -> Result<Array, AfError> {
+    let dims_err = input.dims();
+    let dims = match dims_err {
+        Ok(dim) =>  dim.clone(),
+        Err(e)  =>  panic!("Error unwrapping dims in rows(): {}", e),
+    };
+
+    index(input, &[Seq::new(first as f64, last as f64, 1.0)
+                    ,Seq::new(0.0, dims[1] as f64 - 1.0, 1.0)])
```

Comment:
Should be fine i think, `index` call will return appropriate error code if something is wrong. I don't think we should duplicate error checks in rust wrapper unless absolutely required.